haskell-src/Concordium/Types/Queries/Tokens.hs

```diff
@@ -1,11 +1,13 @@
 {-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE OverloadedStrings #-}
 
 -- | Types for protocol level tokens (PLT).
 module Concordium.Types.Queries.Tokens where
 
-import Data.Word
+import qualified Concordium.Types as Types
 
-import Concordium.Types
+import Data.Word
```

Comment:
Style: Generally, we group imports as:
- External imports
- Imports from `concordium-base`
- Imports from the package (e.g. `concordium-node`)